cluster-diagnosis/utils.py

```diff
@@ -16,6 +16,8 @@
 import sys
 import subprocess
 import logging
+import time
+import collections
```

Comment:
Nitpick: reorder the imports to keep in lexicographical order.

Reply:
Done.